dev-libs/gjs/gjs-1.73.2.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic gnome.org meson virtualx
+
+DESCRIPTION="Javascript bindings for GNOME"
+HOMEPAGE="https://wiki.gnome.org/Projects/Gjs https://gitlab.gnome.org/GNOME/gjs"
+
+LICENSE="MIT || ( MPL-1.1 LGPL-2+ GPL-2+ )"
+SLOT="0"
+IUSE="+cairo examples gtk readline sysprof test"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.66.0:2
+	dev-libs/libffi:=
+	>=dev-libs/gobject-introspection-1.66.1:=
+	>=dev-lang/spidermonkey-91.3.0:91
```

Comment:
gjs-1.73.2 depends on mozjs-102

https://github.com/GNOME/gjs/blob/1.73.2/meson.build#L130

Reply:
Damn, I thought it did this. Must have slipped though with all the undos and redos and such.

Thanks!